js/src/Ice/Reference.js

```diff
@@ -948,7 +948,6 @@ export class RoutableReference extends Reference {
         const properties = new Map();
 
         properties.set(prefix, this.toString());
-        properties.set(prefix + ".CollocationOptimized", "0");
```

Comment:
Since properties are now language-specific, we want to get rid of .CollocationOptimized in JS.